src/shared/components/executiveStaffCard/executiveStaffCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './executiveStaffCard.css';
+
+const ExecutiveStaffCard = ({ name, role, slack, email }) => (
```

Comment:
Frankly, now that I'm thinking about it... boardCard, teamCard, executiveStaff card should all just be boiled down to staffCard and there should be props to determine whether or not functions like a regular team card or whether or not in functions like a boardMember card.